### PR TITLE
WEBAPP-137: modifications to support new People Pattern pipeline, whi…

### PR DESCRIPTION
**Pull request by scottmartinnet**
…ch processes Actors.

### CODE REVIEW

File: components/streams-processor-peoplepattern/pom.xml [hunk 1]
```diff
@@ -21,13 +21,15 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>com.peoplepattern</groupId>
-        <artifactId>components</artifactId>
-        <version>0.3-incubating-SNAPSHOT</version>
+        <groupId>org.apache.streams</groupId>
+        <artifactId>streams-project</artifactId>
+        <version>0.3.1-PP-SNAPSHOT</version>
```
Comment (steveblackmon) [inline components/streams-processor-peoplepattern/pom.xml]: with a parent from com.peoplepattern no one outside our circle of trust will be able to build this module.

master branch should probably have parent org.apache.streams:streams-project:0.2-incubating
or org.apache.streams:streams-project:0.3-incubating-SNAPSHOT
if using the most recent release isn't workable.

if org.apache.streams:streams-project:0.3-incubating-SNAPSHOT lacks critical changes 
made to org.apache.streams:streams-project:0.3.1-PP-SNAPSHOT, changes in peoplepattern/streams 
that require those changes should not merge to master until upstream dependencies have merged.

File: components/streams-processor-peoplepattern/pom.xml [hunk 2]
```diff
@@ -35,6 +37,7 @@
         <dependency>
             <groupId>org.apache.streams</groupId>
             <artifactId>streams-config</artifactId>
+            <version>0.3.1-PP-SNAPSHOT</version>
```
Comment (steveblackmon) [inline components/streams-processor-peoplepattern/pom.xml]: these version declarations should probably be dropped (after parent switch-over), or replaced with ${streams.version} declared in a <properties> section in the top level pom

**Comment by scottmartinnet** [comment]
Changes made.

**Comment by steveblackmon** [comment]
:+1: merging

**Pull request merged**